### PR TITLE
customize robots.txt file

### PR DESCRIPTION
**Pull request by sabinem**

### CODE REVIEW

File: ckanext/switzerland/templates/robots.txt [hunk 1]
```diff
@@ -0,0 +1,4 @@
+User-agent: *
+{% block all_user_agents -%}
```
Comment (stefina) [inline ckanext/switzerland/templates/robots.txt:2]: AFAIK the syntax for a block is `{% block here_comes_the_block_name %}`
Can you double-check the syntax here and in the `{%- endblock %}`-line? 
If I remember it correctly, it should be possible to acess the robots.txt locally, too

**Review by stefina — CHANGES_REQUESTED** [review]

File: ckanext/switzerland/templates/robots.txt (hunk 1 above)
Comment (sabinem) [reply ckanext/switzerland/templates/robots.txt:2]: I copied the template over from ckan and just changed the line in the block, see here: https://github.com/ckan/ckan/blob/2.8/ckan/templates/robots.txt

Comment (stefina) [reply ckanext/switzerland/templates/robots.txt:2]: okay, did not remember that. if it works for you locally, I think we can merge it :)

Comment (sabinem) [reply ckanext/switzerland/templates/robots.txt:2]: Okay thanks.

**Pull request merged**